components/ButtonBar/src/private/MDCAppBarButtonBarBuilder.m

```diff
@@ -54,7 +54,25 @@ @interface UIBarButtonItem (MDCHeaderInternal)
 
 @end
 
-@implementation MDCAppBarButtonBarBuilder
+@implementation MDCAppBarButtonBarBuilder {
+  NSMutableDictionary<NSNumber *, UIFont *> *_fonts;
+}
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    _fonts = [NSMutableDictionary dictionary];
+  }
+  return self;
+}
+
+- (nullable UIFont *)titleFontForState:(UIControlState)state {
+  return _fonts[@(state)];
```

Comment:
Is this expected to fallback to `.normal` like many of our other forState: APIs?

Reply:
Ah this was modeled after MDCButton, but it sounds like MDCButton's behavior is incorrect?